cluster-provision/gocli/cmd/run.go

```diff
@@ -576,15 +585,22 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
 			return err
 		}
 
+		if !success {
+			return fmt.Errorf("checking for ssh.sh script for node %s failed", nodeName)
+		}
+
+		success, err = docker.Exec(cli, nodeContainer(prefix, nodeName), []string{"/bin/bash", "-c", "scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i vagrant.key " + fmt.Sprintf("-r /scripts vagrant@192.168.66.10%d:/home/vagrant/scripts", x+1)}, os.Stdout)
+		if err != nil {
+			return err
+		}
+
+		if !success {
+			return fmt.Errorf("Copying scripts directory for node %s failed", nodeName)
+		}
```

Comment:
This seems to appear here from nowhere.  Please elaborate why/how...

Reply:
if you think this comment is enough please resolve the conversation